src/go/rpk/pkg/cli/cmd/cluster/config/export/export.go

```diff
@@ -0,0 +1,205 @@
+// Copyright 2021 Vectorized, Inc.
+//
+// Use of this software is governed by the Business Source License
+// included in the file licenses/BSL.md
+//
+// As of the Change Date specified in that file, in accordance with
+// the Business Source License, use of this software will be governed
+// by the Apache License, Version 2.0
+
+package export
+
+import (
+	"fmt"
+	"io/ioutil"
+	"os"
+	"sort"
+
+	log "github.com/sirupsen/logrus"
+	"github.com/spf13/afero"
+	"github.com/spf13/cobra"
+	"github.com/vectorizedio/redpanda/src/go/rpk/pkg/api/admin"
+	"github.com/vectorizedio/redpanda/src/go/rpk/pkg/config"
+	"github.com/vectorizedio/redpanda/src/go/rpk/pkg/out"
+	yaml "gopkg.in/yaml.v3"
+)
+
+func ExportConfig(
```

Comment:
Am I reading this function correctly that the purpose of it is to transform the configuration returned by redpanda's new /v1/cluster_config/schema api into a yaml file?

I may have an alternative idea to make this simpler if so (may, ... maybe not though). As well, it may be worth it to support multiple export/import options, but that's not necessary for this PR (toml).

Reply:
Yes, it is essentially a presentation layer for the machine readable config.

This isn't meant to be a widely used command -- it's mainly there because:
- it's easy, as it's just a reused component of the primary `edit` user flow
- it enables scripting for anyone who wants to do an export/sed/import cycle in a script (although obvs I would prefer those people just use the API)